### PR TITLE
Minor edits, add more links to resources

### PR DESCRIPTION
**Pull request by NukeManDan**
- Link to full STARK math series
- Link to video series (https://starkware.co/developers-community/stark101-onlinecourse/)
- Formatting more latex-like
- Update notebooks to link to videos and slides
- add `.editorconfig` for use in formatting style (automatically in IDE)

---
This change is [Reviewable](https://reviewable.io/reviews/starkware-industries/stark101/6)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,13 +26,16 @@ For the tutorial we define a sequence that resembles the well known Fibonacci se
...
 
 ### Some Other Things You Should Know
-We recommend you take a look at our math blog posts ([Arithmetization I](https://medium.com/starkware/arithmetization-i-15c046390862), [Arithmetization II](https://medium.com/starkware/arithmetization-ii-403c3b3f4355)). You don’t need to read them thoroughly before running through the tutorial, but it can give you better context on what things you can create proofs for, and what a STARK proof looks like. You should definitely give them a read after you have completed this tutorial in full.
+We recommend you take a look at our [STARK math blog posts](https://medium.com/starkware/tagged/stark-math) (Arithmetization [I](https://medium.com/starkware/arithmetization-i-15c046390862) & [II](https://medium.com/starkware/arithmetization-ii-403c3b3f4355) specifically). You don’t need to read them thoroughly before running through the tutorial, but it can give you better context on what things you can create proofs for, and what a STARK proof looks like. You should definitely give them a read after you have completed this tutorial in full.
```
Comment (NukeManDan) [inline README.md:33]: only edit of note

**Comment by NukeManDan** [comment]
I would like to add a link to a full (public) playlist for these videos. they are all on youtube, but unlisted (https://www.youtube.com/watch?v=Y0uJz9VL3Fo is the first).
Can these be publicly listed and a playlist created by the official channel?

**Review by liorgold2 — CHANGES_REQUESTED** [review]
> Reviewed 6 of 6 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-industries/stark101/6)* status: all files reviewed, 4 unresolved discussions (waiting on @NukeManDan)

___
*[README.md, line 4 at r2](https://reviewable.io/reviews/starkware-industries/stark101/6#-Mia0n3E5kNdS0XvX0OT:-Mia0n3E5kNdS0XvX0OU:bhsdzlh) ([raw file](https://github.com/starkware-industries/stark101/blob/4351d3fbf4f24b7d43b19392f4252550fa441c30/README.md#L4)):*
> ```Markdown
> 
> ## About
> A tutorial for a basic STARK (***S**calable **T**ransparent **AR**gument of **K**nowledge) protocol to prove the calculation of a Fibonacci-Square sequence, as designed for [StarkWare Sessions](https://starkware.co/starkware-sessions/), and authored by the [StarkWare](https://starkware.co) team.
> ```

Fix `***` -> `**`
___
*[README.md, line 9 at r2](https://reviewable.io/reviews/starkware-industries/stark101/6#-Mia0vX0CFMudlZ6iJx6:-Mia0vX0CFMudlZ6iJx7:b-cjiu9d) ([raw file](https://github.com/starkware-industries/stark101/blob/4351d3fbf4f24b7d43b19392f4252550fa441c30/README.md#L9)):*
> ```Markdown
> ### Video Lessons
> 
> This tutorial has a **complimentary series of videos [available here](https://starkware.co/developers-community/stark101-onlinecourse/)** to review. Slides and links (to this repositories content) are also included.
> ```

Remove the word "complimentary".
I know the current file doesn't follow this rule because it is old, but please split new lines to up to 100 chars per line.
___
*[README.md, line 51 at r2](https://reviewable.io/reviews/starkware-industries/stark101/6#-Mia4h5u8ihrvfBuEsFO:-Mia4h5u8ihrvfBuEsFP:b-y32n5y) ([raw file](https://github.com/starkware-industries/stark101/blob/4351d3fbf4f24b7d43b19392f4252550fa441c30/README.md#L51)):*
> ```Markdown
> 1. Nigel Smart’s [“Cryptography Made Simple”](https://www.cs.umd.edu/~waa/414-F11/IntroToCrypto.pdf) – Chapter 1.1: Modular Arithmetic.
> 
> 2. Arora and Barak’s [“Computational Complexity: A Modern Approach”](http://theory.cs.princeton.edu/complexity/book.pdf) – Appendix: Mathematical Background, sections A.4 (Finite fields and Groups) and A.6 (Polynomials).
> ```

Please end the file with the new-line character.
___
*[tutorial/Stark101-part5.ipynb, line 1 at r2](https://reviewable.io/reviews/starkware-industries/stark101/6#-Mia5E928olcH5xhvV21:-Mia5E928olcH5xhvV22:bkm6e5y) ([raw file](https://github.com/starkware-industries/stark101/blob/4351d3fbf4f24b7d43b19392f4252550fa441c30/tutorial/Stark101-part5.ipynb#L1)):*
> ```
> {
> ```

Reviewing this will take more time, so I suggest moving it to a different PR, so that we can merge this one.

**Comment by NukeManDan** [comment]
Formatting as suggested, Part 5 is _only_ a video recap mention. The file can be removed, but not sure where else to put it to follow the https://starkware.co/developers-community/stark101-onlinecourse/ outline 1 to 1.

**Review by liorgold2 — CHANGES_REQUESTED** [review]
> Reviewed 2 of 3 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-industries/stark101/6)* status: 6 of 7 files reviewed, 3 unresolved discussions (waiting on @liorgold2 and @NukeManDan)

___
*[.editorconfig, line 1 at r3](https://reviewable.io/reviews/starkware-industries/stark101/6#-MijvbCkE-vctaSBL_YX:-MijvbCkE-vctaSBL_YY:b-nzeoc5) ([raw file](https://github.com/starkware-industries/stark101/blob/1854099b1fbfd21c66ad8520999b30768fc32d9a/.editorconfig#L1)):*
> ```EditorConfig
> root = true
> ```

Should this file be part of the git?
___
*[README.md, line 9 at r2](https://reviewable.io/reviews/starkware-industries/stark101/6#-Mia0vX0CFMudlZ6iJx6:-MijwYwl8ePBfttcPoLm:b-ucfknh) ([raw file](https://github.com/starkware-industries/stark101/blob/4351d3fbf4f24b7d43b19392f4252550fa441c30/README.md#L9)):*
<details><summary><i>Previously, liorgold2 wrote…</i></summary><blockquote>

Remove the word "complimentary".
I know the current file doesn't follow this rule because it is old, but please split new lines to up to 100 chars per line.
</blockquote></details>

Thanks!
___
*[README.md, line 11 at r3](https://reviewable.io/reviews/starkware-industries/stark101/6#-MijvmGuEIF5n7ObsAmF:-MijvmGuEIF5n7ObsAmG:bd50422) ([raw file](https://github.com/starkware-industries/stark101/blob/1854099b1fbfd21c66ad8520999b30768fc32d9a/README.md#L11)):*
> ```Markdown
> 
> Note that it was written assuming that the user has reviewed and understood the presentations at the
> beginning of each part. 
> ```

Remove the two spaces at the end of the line.

**Review by liorgold2 — COMMENTED** [review]
> Reviewed 1 of 3 files at r3.
> *[Reviewable](https://reviewable.io/reviews/starkware-industries/stark101/6)* status: all files reviewed, 2 unresolved discussions (waiting on @NukeManDan)

___
*[tutorial/Stark101-part5.ipynb, line 1 at r2](https://reviewable.io/reviews/starkware-industries/stark101/6#-Mia5E928olcH5xhvV21:-MijwrJK2ihL8Z6zZ7Ub:b-eapff3) ([raw file](https://github.com/starkware-industries/stark101/blob/4351d3fbf4f24b7d43b19392f4252550fa441c30/tutorial/Stark101-part5.ipynb#L1)):*
<details><summary><i>Previously, liorgold2 wrote…</i></summary><blockquote>

Reviewing this will take more time, so I suggest moving it to a different PR, so that we can merge this one.
</blockquote></details>

Resolved.

**Review by liorgold2 — COMMENTED** [review]
I see. Cool. Keep part 5 in this PR, then.
Thanks!

> *[Reviewable](https://reviewable.io/reviews/starkware-industries/stark101/6#-:-Mijwy2t-Kk5shYVIwMX:bwznhj5)* status: all files reviewed, 2 unresolved discussions (waiting on @NukeManDan)

**Comment by NukeManDan** [comment]
> Should this file be part of the git?
`.editorconfig` helps _most_ editors to comply with your required formatting. This is optional for people to use, but expresses what _should_ be used if not automatically used (by extensions like Prettier). I would suggest adding it. Many docs sites do this as well from my experience, and I find it help full. 

I think with that we are all good to go? 🤞🏼

**Comment by NukeManDan** [comment]
Ah and also thie includes #5 , so that can be safely closed in favor of this PR.

**Review by liorgold2 — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-industries/stark101/6#-:-MiuRoCeB5kRqaePQBN5:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @NukeManDan)

**Review by liorgold2 — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/starkware-industries/stark101/6)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @NukeManDan)

**Comment by NukeManDan** [comment]
Awesome, just needs merged? I don't have the power to.

**Pull request merged**